lighthouse-core/audits/byte-efficiency/uses-optimized-images.js

```diff
@@ -41,7 +40,7 @@ class UsesOptimizedImages extends Audit {
     return {
       category: 'Images',
       name: 'uses-optimized-images',
-      description: 'Has optimized images',
+      description: 'Avoids unoptimized images',
```

Comment:
I switched them because I sympathized with the idea that telling users they did something when that thing doesn't apply to them doesn't make a lot of sense. I like the affirmative descriptions in isolation, but I do think consistency between audits within a category that's all about *not* doing something (shipping bytes) feels a lot smoother.

I definitely prefer "Optimizes images" to the double negative here, but could use guidance on the CSS affirmative, `Uses most of its CSS` is pretty lame